.github/workflows/driver_ci.yaml

```diff
@@ -0,0 +1,201 @@
+name: CSI IBM Driver CI
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+    branches:
+      - develop
+      - master
+jobs:
+  prepare_env:
+   name: "prepare env"
+   runs-on: ubuntu-latest
+   steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+      - name: Setup dependencies
+        run: |
+          scripts/ci/setup_ci_dependencies.sh
+      - uses: actions/cache@v2
+        with:
+          path: ${{ env.pythonLocation }}
+          key: ${{ env.pythonLocation }}-${{ hashFiles('dev-requirements.txt') }}
+      - name: Save dependencies files
+        uses: actions/upload-artifact@v2
+        with:
+          name: ci-dependencies
+          path: |
+            /home/runner/.bash_profile
+          retention-days: 1
+      - uses: FranzDiebold/github-env-vars-action@v2
+  CSI-controller-static_code_analysis:
```

Comment:
![image](https://user-images.githubusercontent.com/57443811/125259679-372ef580-e308-11eb-8fbf-3f396991cd6c.png)
please choose casing + punctuation convention and align all accordingly

Reply:
and align all with the [stage names](https://github.com/IBM/ibm-block-csi-driver/blob/a538b5ae9ee1822ed702671aa09fdbd9965ce7c3/scripts/ci/jenkins_pipeline_csi#L11) accordingly

Reply:
done

Reply:
let me clarify - e.g. `prepare env` and `Create-cluster` are not in the same convention, because of whitespace vs. `-`, uppercase vs. lowercase, etc...

I personally prefer whitespace over `-`, but the main point is for all to be aligned.

btw, I know the example yamls dir is currently under a "deploy" dir, but "k8s yamls validation" is not "CSI deployment", it's more "[CSI configuration](https://www.ibm.com/docs/en/stg-block-csi-driver/1.6.0?topic=csi-driver-configuration)" (the true deployment dir is [on the operator side](https://github.com/IBM/ibm-block-csi-operator/tree/v1.6.0/deploy))

Reply:
I still see "CSI deployment"

Reply:
and `CSI-deployment-k8s_yamls_validation`

Reply:
as I [wrote](https://github.com/IBM/ibm-block-csi-driver/pull/344#discussion_r669604244), IMO, "k8s yamls validation" is sufficient

Reply:
and [again](https://github.com/IBM/ibm-block-csi-driver/pull/344#discussion_r667775663), please align the names with the [stage names](https://github.com/IBM/ibm-block-csi-driver/pull/18/files#diff-66d95ee8647d887bdbe0c60dfaddb85d248f5bb09bc843c3c6e5949463199272R23)

Reply:
Do you want me to change in the Jenkinsfile also?

Reply:
yes, thanks

Reply:
done

Reply:
`prepare env` is still lowercase while the rest are capitalized

Reply:
done

Reply:
the case of the first letter is [still not unified](https://github.com/IBM/ibm-block-csi-driver/blob/3b3c0b9a16f14e4d49be9aa59bd56c9beb8f4881/.github/workflows/driver_ci.yaml#L74) in the job "keys".
also it is not the same as in the [other PR](https://github.com/IBM/ibm-block-csi-operator/blob/aba7bb060c67f23e38e5089145730a6a5b3e84ee/.github/workflows/operator_ci.yaml#L65)

Reply:
I still see [this](https://github.com/IBM/ibm-block-csi-driver/pull/344#discussion_r676344450)

Reply:
I think I got it right now, I think I was confused from all the comments

Reply:
perhaps I wasn't specific enough (though I don't think I should link to every relevant line):
I still see [this](https://github.com/IBM/ibm-block-csi-driver/blob/a05503f9c8f49363ea1e32c16f870f055c2bc857/.github/workflows/driver_ci.yaml#L76)